extensions/http(s)/lib/ace/http/handler.ex

```diff
@@ -84,16 +84,14 @@ defmodule Ace.HTTP.Handler do
         # e.g. localhost:8080//
         path = path || "/"
         {:ok, query} = URI2.Query.decode(query_string || "")
-        path = Raxx.Request.split_path(path)
+        path = split_path(path)
         scheme = case conn_info.transport do
           :tcp -> "http"
           :ssl -> "https" # DEBT remove
           :tls -> "https"
         end
-        peer = conn_info.peer
```

Comment:
This is stuff relating to the details of the tcp connection, stuff like the ip address and port number of the browser. its not availabe in the same way in HTTP/2 so its not added by default to the request